esp8266-fastled-webserver/data/js/simple.js

```diff
@@ -1,10 +1,10 @@
 // used when hosting the site on the ESP8266
-var address = location.hostname;
-var urlBase = "";
+// var address = location.hostname;
+// var urlBase = "";
 
 // used when hosting the site somewhere other than the ESP8266 (handy for testing without waiting forever to upload to SPIFFS/LittleFS)
-// var address = "192.168.1.13";
-// var urlBase = "http://" + address + "/";
+var address = "192.168.86.30";
```

Comment:
I think this was left in by accident?

Reply:
1. Yes, fixed, thanks!
2. All of the glyphicons files have been removed as part of this PR.